Podfile

```diff
@@ -51,7 +51,7 @@ abstract_target 'WordPress_Base' do
     pod 'WPMediaPicker', '0.13'
     pod 'WordPress-iOS-Editor', '1.9.0'
     pod 'WordPressCom-Analytics-iOS', '0.1.23'
-    pod 'WordPress-Aztec-iOS', '0.5a6'
+    pod 'WordPress-Aztec-iOS', :git => 'https://github.com/wordpress-mobile/WordPress-Aztec-iOS.git', :commit => '5ead13b91446d00135d78f01479d36dac6c84f45'
```

Comment:
@astralbodies - Are we sure we want to version a full version for each integration?  That feels like a lot of overhead considering the library is being actively developed alongside WPiOS.

By integrating frequently (without the need to version each time) we're reducing the time it takes to solve dependencies between issues.